app/src/routes/home.tsx

```diff
@@ -148,27 +155,31 @@ const Footer = () => {
   )
 }
 
+
+
 export default () => {
   const fbContext = useContext(FirebaseContext);
-  const [dishesUsed, setDishesUsed] = useState('');
+  const [dishesUsed, setDishesUsed] = useState([]);
   var content;
-  const location = useLocation();
-  let sessionToken = location.state
-  // axios.get('http://localhost:8080/api/transactions', {headers:{"x-api-key":"test","session-token":sessionToken}})
-  // .then(function (response) {
-  //   setDishesUsed(response.data.transactions.length)
-  // })
-  // .catch(function (error) {
-  //   console.log(error);
-  // });
+  let token = SessionToken()
+  console.log(token)
+  // Fetch dishes transaction for the user
+  useEffect(()=>{
+    axios.get('http://ec2-34-213-210-231.us-west-2.compute.amazonaws.com/api/transactions', {headers:{"x-api-key":"test","session-token":token}})
```

Comment:
Should probably put the hostname of your server in an environment variable, eg

```
BACKEND_ADDRESS=http://ec2-34-213-210-231.us-west-2.compute.amazonaws.com
```

and then ```axios.get(`${process.env.BRACKEND_ADDRESS}/api/transactions``` , to avoid typos and make it easier to change in the future